examples/1bitsy/tft-st7735/Makefile

```diff
@@ -0,0 +1,28 @@
+##
+## This file is part of the libopencm3 project.
+##
+## Copyright (C) 2009 Uwe Hermann <uwe@hermann-uwe.de>
+##
+## This library is free software: you can redistribute it and/or modify
+## it under the terms of the GNU Lesser General Public License as published by
+## the Free Software Foundation, either version 3 of the License, or
+## (at your option) any later version.
+##
+## This library is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+## GNU Lesser General Public License for more details.
+##
+## You should have received a copy of the GNU Lesser General Public License
+## along with this library.  If not, see <http://www.gnu.org/licenses/>.
+##
+
+BINARY = st7735lcd
+SOURCES  = systick.c
+OBJS := ../common/button_boot.o $(SOURCES:%.c=%.o)
+
+LDSCRIPT = ../stm32f4-1bitsy.ld
+
+include ../Makefile.include
+
+CFLAGS += -O0
```

Comment:
This should not be necessary, by default GCC does not compile code with optimization. If you want to tell a user that they can enable optimization in this place it would be better to have:
```
# Uncomment the following line to enable size optimization
#CFLAGS += -Os
```

I am not sure if forcing to `-O0` is necessary here.